src/styles/styles.ts

```diff
@@ -0,0 +1,29 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Remove this file, use inline styles or rules from theme.

Reply:
Resolved on c5e00f9c5aabbdf091be97a74849f5cc5605075e